jib-core/src/main/java/com/google/cloud/tools/jib/builder/ProgressEventDispatcher.java

```diff
@@ -147,7 +147,7 @@ public void dispatchProgress(long progressUnits) {
   private long decrementRemainingAllocationUnits(long units) {
     Preconditions.checkState(!closed);
 
-    if (remainingAllocationUnits > units) {
+    if (remainingAllocationUnits >= units) {
```

Comment:
Note: this is unrelated to #1796 and does not change the behavior. (If they are same, the remaining units become 0.) I just think `>=` looks more intuitive.

Reply:
To me `>=` immediately followed by the subtraction looks less intuitive, but it's small.

Reply:
That may be true if you look at this condition and the true branch, but I found it difficult to digest the false branch if the remaining units are equal to the units to subtract.